service/src/main/java/bio/terra/cbas/dependencies/common/ScheduledHealthChecker.java

```diff
@@ -34,7 +34,7 @@ public ScheduledHealthChecker(
       LeonardoService leonardoService,
       SamService samService,
       EcmService ecmService,
-      BardService bardService,
+      @Qualifier("bardServiceImpl") HealthCheck bardService,
```

Comment:
If `BardService` implemented HealthCheck too, I think you could avoid doing it like this

Reply:
Oh interesting...I didn't think about making the `BardService` interface implement the `HealthCheck` interface